config/prow/plugins.yaml

```diff
@@ -246,36 +246,36 @@ slack:
     - k8s-ci-robot # future home of tide
     - k8s-release-robot # anago
     exempt_branches:
-        release-1.17:
```

Comment:
I think we need to keep this for now, since 1.17 is not deprecated yet. not 100% sure

Reply:
I've thought the same, but the [branch management handbook](https://github.com/kubernetes/sig-release/blob/master/release-engineering/role-handbooks/branch-manager.md#update-slack-branch-whitelists) says:
> Remove the oldest release branch block

I'd say that we leave 1.17 and remove it at some point later, but I'm not sure is that okay.

Reply:
Yep, end of Life for 1.17 is 2021-01-30

Reply:
@cpanato @saschagrunert PTAL, updated.